apps/zbugs/server/push-handler.ts

```diff
@@ -1,19 +1,13 @@
-import {
-  ZQLDatabase,
-  PostgresJSConnection,
-  PushProcessor,
-} from '@rocicorp/zero/pg';
+import {PushProcessor} from '@rocicorp/zero/server';
+import {zeroPostgresJS} from '@rocicorp/zero/server/adapters/postgresjs';
 import postgres from 'postgres';
 import {schema} from '../shared/schema.ts';
 import {createServerMutators, type PostCommitTask} from './server-mutators.ts';
 import type {AuthData} from '../shared/auth.ts';
 import type {ReadonlyJSONValue} from '@rocicorp/zero';
 
 const processor = new PushProcessor(
-  new ZQLDatabase(
-    new PostgresJSConnection(postgres(process.env.ZERO_UPSTREAM_DB as string)),
-    schema,
-  ),
+  zeroPostgresJS(schema, postgres(process.env.ZERO_UPSTREAM_DB as string)),
```

Comment:
If we are making this tied to `postgres` npm package, can we put that inside `zeroPostgresJS`?